github-actions/index.html

```diff
@@ -0,0 +1,158 @@
+<!doctype html>
+<html class="no-js" lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>University of Illinois GitHub Service Community Portal: GitHub Actions</title>
+  <meta name="description" content="Community-supported documentation for the University of Illinois GitHub service.">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="robots" content="noindex, nofollow">
+
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
+    integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+  <link rel="styleheet" href="/css/illinois-system.css">
+
+  <meta name="theme-color" content="#fafafa">
+</head>
+
+<body>
+  <header class="blog-header my-3 pb-3 border-bottom">
+    <div class="container">
+      <h1>University of Illinois GitHub Service Community Portal - GitHub Actions</h1>
+    </div>
+  </header>
+  <main class="container">
+    <div class="jumbotron rounded-0">
```

Comment:
I'm not sure the jumbotron is doing readability any favors. You've already got the blog header to introduce the page. At the very least, I would trim this down to a quick introductory paragraph, and leave the meat of the page in a regular container.

Reply:
Yanked out the jumbotron entirely. Tried a quick intro and given how short the page was it didn't seem worth the extra focus.